src/lib/realmdb/RealmDB.js

```diff
@@ -196,8 +199,14 @@ class RealmDB implements DB, ProfileDB {
    * used in Appswitch to sync with remote when user comes back to app
    */
   async _syncFromRemote() {
+    const { _mutex } = this
```

Comment:
@roma-hladilka 

**TOTALLY WRONG!**

1. i've asked to dedicate mutex per each source
2. no error handling - mutex won't be released if error happens!